pycode/test/test_epidata_getCommuterMobility.py

```diff
@@ -222,90 +263,140 @@ def write_commuter_all_federal_states(self, out_folder):
             for sheet_name in sheets.keys():
                 sheets[sheet_name].to_excel(dummy, sheet_name=sheet_name, index=False)
             dummy.save()
+            dummy.close()
+
+    @patch('builtins.print')
+    def test_verify_sorted(self, mock_print):
+        self.assertEqual(True, gcm.verify_sorted(self.test_countykey_list))
+        self.assertEqual(False, gcm.verify_sorted(self.test_countykey_list2))
+        Errorcall = ('Error. Input list not sorted, population per county list had to '
+                     'be sorted accordingly.')
+        mock_print.assert_called_with(Errorcall)
+        # test case with empty list
+
+        gcm.verify_sorted(())
```

Comment:
See my comment in the function itself. This test should be adapted/removed accordingly